openmeter/ent/schema/productcatalog.go

```diff
@@ -0,0 +1,251 @@
+package schema
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+
+	"entgo.io/ent"
+	"entgo.io/ent/dialect"
+	"entgo.io/ent/dialect/entsql"
+	"entgo.io/ent/dialect/sql"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+
+	"github.com/openmeterio/openmeter/openmeter/productcatalog/plan"
+	"github.com/openmeterio/openmeter/pkg/datex"
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+type Plan struct {
+	ent.Schema
+}
+
+func (Plan) Mixin() []ent.Mixin {
+	return []ent.Mixin{
```

Comment:
we have the `UniqueResourceMixin` to combine these:
https://github.com/openmeterio/openmeter/blob/28e9c4d5535a52df6c79af171ea8229c4c9c8ca8/pkg/framework/entutils/mixins.go#L18

Reply:
Done.